index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="css/style.css" />
+	<script src="https://kit.fontawesome.com/102965b95f.js" crossorigin="anonymous"></script>
+	<title>Document</title>
+</head>
+
+<body>
+	<header class="header">
+		<nav class="bar">
+			<ul class="bar__cont">
+				<li class="bar__item"> <a class="bar__item" href="#"><i class="fa fa-twitter"></i> </a></li>
+				<li class="bar__item"><a class="bar__item" href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="bar__item"><a class="bar__item" href="#">English</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">My Page</a></li>
+				<li class="bar__item"><a class="bar__item" href="#">Logout</a></li>
+			</ul>
+		</nav>
+		<i class="header__menu fa fa-bars fa-2x"></i>
+		<nav class="header__nav">
+			<a href="#home" class="header__home"> <img src="images/LogoMakr-7y5JLd.png" alt="Faith Apostolic logo"
+					class="header__logo"></a>
+
+			<ul class="header__nav--cont">
+				<li class="header__nav--item"><a href="#program">Program</a></li>
+				<li class="header__nav--item"><a href="about.html">About</a></li>
+				<li class="header__nav--item"><a href="speakers">Speakers</a></li>
+			</ul>
+			<button type="button" class="header__camp">Campaign</button>
+		</nav>
+		<div class="mobile-menu" id="mobile-menu">
+			<button type="button" class="mobile-close">&times;</button>
+			<ul class="menu-list">
+				<li class="list-item"> <a href="#program">Program</a> </li>
+				<li class="list-item"><a href="about.html">About</a></li>
+				<li class="list-item"><a href="#speakers">Speakers</a></li>
+			</ul>
+		</div>
+	</header>
+	<div class="container1" id="home">
```

Comment:
- [ ] You should align this section correctly. In the design template, it has more top margin and less left margin than yours. Please compare the given screenshots.

Your Project :arrow_down: 
![Screenshot 2021-07-01 at 23-31-57 Document](https://user-images.githubusercontent.com/1642014/124188760-5dfb5980-dac8-11eb-902a-fcbe7a9b9281.png)

Design Template :arrow_down: 
![Screenshot from 2021-07-01 20-43-02](https://user-images.githubusercontent.com/1642014/124188781-62c00d80-dac8-11eb-9b79-ea76dcbc0f05.png)